charts/placement-policy-scheduler-plugins/templates/rbac.yaml

```diff
@@ -71,9 +90,37 @@ subjects:
   apiGroup: rbac.authorization.k8s.io
 roleRef:
   kind: ClusterRole
-  name:  pp-plugins-scheduler
+  name:  system:pp-plugins-scheduler
   apiGroup: rbac.authorization.k8s.io
 ---
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name:  pp-plugins-scheduler-auth-delegator
+subjects:
+- kind: User
+  name: system:kube-scheduler
+  namespace: {{ .Release.Namespace }}
+  apiGroup: rbac.authorization.k8s.io
+roleRef:
+  kind: ClusterRole
+  name:  pp-plugins-scheduler:system:auth-delegator
```

Comment:
Where is this cluster role created? I don't see a `ClusterRole` with `pp-plugins-scheduler:system:auth-delegator`.

Reply:
line #113 https://github.com/Azure/placement-policy-scheduler-plugins/pull/53/files/96552457ed96aacd9fe0ab2b49f71c911bb2e491#diff-f6808d3c7a229616496fdefa6c250141964e3c33cf519b9d2aa7ade1752bbb9dR113

Reply:
That's a `ClusterRoleBinding`. This roleRef is referring to a `ClusterRole`.

Reply:
removed, unused